src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import cn from 'classnames';
+
+import './Filter.scss';
+
+type Props = {
+  todoFilter: string;
+  handleFilter: React.Dispatch<React.SetStateAction<string>>;
+};
+
+export const Filter: React.FC<Props> = ({
+  todoFilter,
+  handleFilter,
+}) => {
+  return (
+    <nav className="filter">
+      <a
+        href="#/"
+        className={cn('filter__link', { selected: todoFilter === '' })}
+        onClick={() => handleFilter('')}
+      >
+        All
+      </a>
+
+      <a
+        href="#/active"
+        className={cn('filter__link', { selected: todoFilter === 'active' })}
+        onClick={() => handleFilter('active')}
```

Comment:
consider to make handler factory for all sort fields and just call with proper filter in jsx
```
const makeHandleFilter = (sortField) => () => handleFilter(sortFiled)
```
```suggestion
        onClick={makeHandleFilter('active')}
```